db/emoji_reactions.py

```diff
@@ -49,3 +52,86 @@ def get_reactions_for_user(user_id: int, session: sessionmaker) -> list[str]:
         ).all()
 
         return [row[0].emoji for row in results]
+    
+def get_emoji_reaction_delay(session: sessionmaker) -> int:
+    """
+    Get Robomoji delay time 
+
+    Returns:
+        int: Delay time in seconds for Robomojis
+    """
+    with session() as sess:
+        result = sess.query(EmojiReactionDelay).first()
+
+        if result is None:
+            sess.execute(insert(EmojiReactionDelay).values(delay_in_seconds=DEFAULT_EMOJI_REACTION_DELAY)) # Fallback default value of 15 seconds
+            return DEFAULT_EMOJI_REACTION_DELAY
+        
+        return result.delay_in_seconds
+
+def set_emoji_reaction_delay(delay_time: int, session: sessionmaker) -> int:
+    """
+    Set Robomoji delay time 
+
+    Args:
+        delay_time (int): Delay time in seconds for Robomojis
+    """
+    with session() as sess:
+        result = sess.query(EmojiReactionDelay).first()
+
+        if result is None:
+            sess.execute(insert(EmojiReactionDelay).values(delay_in_seconds=delay_time))
+            return delay_time
+
+        existing_row: EmojiReactionDelay = result
+        sess.execute(
+            update(EmojiReactionDelay)
+            .where(EmojiReactionDelay.id == existing_row.id)
```

Comment:
nit: I'm pretty sure you can leave off the `.where` here since we only have one row. I haven't tested it with SQLAlchemy so could be wrong, but SQL definitely will let you just `UPDATE emoji_reaction_delay (delay_in_seconds) VALUES ...`